tests/test_nsga2_multi_objective.py

```diff
@@ -16,6 +16,7 @@
 from testing_utils import OptTest
 
 
+@unittest.skip("skip flaky NSGA2 tests")
```

Comment:
I think it's only the `n_obj == 2` test that segfaults, so maybe we should just skip that one?